todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/taskdetail/TaskDetailFragment.java

```diff
@@ -45,9 +45,9 @@
  */
 public class TaskDetailFragment extends Fragment implements TaskDetailContract.View {
 
-    public static final String ARGUMENT_TASK_ID = "TASK_ID";
+    private static final String ARGUMENT_TASK_ID = "TASK_ID";
```

Comment:
Yep. Ran both unit and UI tests. Both of them were only used in this class. `ARGUMENT_TASK_ID` was used in the `newInstance` method. So never outside the class.